server/logger.js

```diff
@@ -0,0 +1,20 @@
+class Logger {
```

Comment:
I might simplify it like this:

```js
const timeStamp = () => '[' + new Date().toISOString() + ']';

const logger = {
  log: console.log.bind(null, timeStamp()),
  warn: console.warn.bind(null, timeStamp()),
  error: console.error.bind(null, timeStamp()),
  debug: console.debug.bind(null, timeStamp()),
}

module.exports = logger;
```

But, I don't think it really matters too much.

Reply:
+, implemented with `.call` though so timestamp value updates